Userland/Libraries/LibCore/EventLoopImplementationUnix.cpp

```diff
@@ -1,4 +1,4 @@
-/*
+#/*
```

Comment:
Not sure what this # is doing here, seems a typo?

Reply:
Sorry, fixed it